lib/osf-components/addon/components/contributors/manager/component.ts

```diff
@@ -50,6 +80,19 @@ export default class ContributorsManager extends Component {
         }
     });
 
+    @task({ withTestWaiter: true, enqueue: true })
+    fetchUsers = task(function *(this: ContributorsManager) {
+        const currentPageResult = yield this.store.query('user', {
```

Comment:
Should we consider using user search endpoint?

Reply:
We discussed, and are going to make that a separate ticket to keep from holding up this PR any more.